platform/src/common/components/Charts/components/index.jsx

```diff
@@ -19,44 +19,123 @@ export const truncate = (str) => {
   return str.length > 20 ? str.substr(0, 20 - 1) + '...' : str;
 };
 
+const pollutantRanges = {
+  pm2_5: [
```

Comment:
Hi @OchiengPaul442 , why are we still "hard coding" here?
Can we try to avoid hard coding since this will definitely create more work for us in the near future?